setup.cfg

```diff
@@ -300,6 +300,9 @@ ignore_missing_imports = True
 [mypy-_strptime.*]
 ignore_missing_imports = True
 
+[mypy-hypothesis.*]
+ignore_missing_imports = True
+
```

Comment:
equal to setup in [open-autonomy setup.cfg](https://github.com/valory-xyz/open-autonomy/blob/8a9a9e33c6b296bd7f54d5ebeba4334ae8a3d8e8/setup.cfg#L131-L132)